execution-environment.yml

```diff
@@ -0,0 +1,12 @@
+---
+version: 1
+dependencies:
+  system: bindep.txt
+
+additional_build_steps:
+  prepend: |
+    RUN pip3 install --upgrade pip setuptools
+    RUN dnf install https://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm -y
+    RUN dnf install ovirt-ansible-collection -y
+  append:
+    - RUN ls -la /usr/share/ansible/collections/ansible_collections/
```

Comment:
https://ansible-builder.readthedocs.io/en/latest/collection_metadata.html

Collections can put metadata of this form into the collection, but it needs to be in `meta/execution-environment.yml`. However, the bigger problem here is that only a few things are allowed to be customized by the collections themselves (as opposed to the user's definition), and prepend/append steps are not allowed.

Collections can put their own dnf requirements in a `bindep.txt` file, but that doesn't solve the main problem here - which is the need for a custom repo. See https://github.com/ansible/ansible-builder/issues/191